frontend/src/pages/chat/components/chat-footer/chat-footer.tsx

```diff
@@ -0,0 +1,48 @@
+import { IconButton } from '#libs/components/icon-button/icon-button.js';
+import { Input } from '#libs/components/input/input.js';
+import { DEFAULT_INPUT } from '#libs/constants/constants.js';
+import { useAppForm, useCallback } from '#libs/hooks/hooks.js';
+import { type ChatInputValue } from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onSend: ({ text }: { text: string }) => void;
+};
+
+type HandleForm = () => void;
+
+const ChatFooter: React.FC<Properties> = ({ onSend }) => {
+  const { control, handleSubmit, errors, reset } =
+    useAppForm<ChatInputValue>(DEFAULT_INPUT);
+
+  const onSubmit = useCallback(
+    ({ text }: ChatInputValue): void => {
+      onSend({ text });
+      reset();
+    },
+    [onSend, reset],
+  );
+
+  return (
+    <footer className={styles['chat-footer']}>
+      <form
+        className={styles['form']}
+        onSubmit={handleSubmit(onSubmit) as HandleForm}
```

Comment:
```suggestion
        onSubmit={handleSubmit(onSubmit)}
```

Reply:
This advice from lint prevents me from committing changes: 

<img width="588" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/a4bb51d2-9ad8-4c59-be28-c54b68f21359">

Reply:
We were creating another useCallback function before, for instance `handleFormSubmit`, thats contains:
```
void handleSubmit(onSubmit)(event_)
```